### PR TITLE
Injiver 762 implementing CAR branding

### PR DESCRIPTION
**Pull request by srikanth716**

### CODE REVIEW

File: ui/src/assets/inji-verify.svg
Comment (sree96) [inline ui/src/assets/inji-verify.svg:1]: Is this still used for this?

File: ui/src/assets/logo.svg
Comment (sree96) [inline ui/src/assets/logo.svg:1]: Why there was a react logo in older impl?

File: ui/src/assets/under-construction.svg
Comment (sree96) [inline ui/src/assets/under-construction.svg:1]: Does this not have theme changes?

**Review by sree96 — CHANGES_REQUESTED** [review]

File: ui/src/assets/inji-verify.svg
Comment (srikanth716) [reply ui/src/assets/inji-verify.svg:1]: Yes, On copyrights section

File: ui/src/assets/logo.svg
Comment (srikanth716) [reply ui/src/assets/logo.svg:1]: It was on part of creating the react app, I think they forgot to remove after.

File: ui/src/assets/under-construction.svg
Comment (srikanth716) [reply ui/src/assets/under-construction.svg:1]: I have missed to move this in theme customization.

File: ui/src/assets/logo.svg
Comment (sree96) [reply ui/src/assets/logo.svg:1]: This means main branch will also have this unused asset also. Lets remove it in next PR

**Review by sree96 — APPROVED** [review]

**Pull request merged**